<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QWidget" name="LoginForm">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>274</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>黄蜂套利交易系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/image/bee.ico</normaloff>:/image/bee.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox_login_form">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>379</width>
     <height>253</height>
    </rect>
   </property>
   <property name="title">
    <string>登录</string>
   </property>
   <widget class="QPushButton" name="pushButton_login">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>110</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>110</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
    <property name="shortcut">
     <string>Esc</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trader_id">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>交易员账户</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_trader_id">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>130</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_trader_password">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>交易员密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_trader_password">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>130</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_isoffline">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>87</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>脱机登录</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_login_error">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>247</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_trader_id</tabstop>
  <tabstop>lineEdit_trader_password</tabstop>
  <tabstop>checkBox_isoffline</tabstop>
  <tabstop>pushButton_login</tabstop>
  <tabstop>pushButton_cancel</tabstop>
 </tabstops>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
